todo-app/src/App.js

```diff
@@ -1,10 +1,86 @@
 import React, { Component } from 'react';
+import Header from './Components/Header'
+import Footer from './Components/Footer'
+import Container from './Components/Container'
+
+
+import './App.css';
+const data = require('./todoList.json');
 
 class App extends Component {
+  state = {
+    todoItems: []
+  }
+  componentWillMount() {
+    localStorage.getItem('todoItems') && this.setState({
+      todoItems: JSON.parse(localStorage.getItem('todoItems'))
+    })
+  }
+  componentDidMount() {
+    if (!localStorage.getItem('todoItems')) {
```

Comment:
Why are you setting state in `componentWillMount` _and_ `componentDidMount`? This is going to cause several renders to happen for no reason. You should initialise your state inside your class property instead of using `setState` in mounting lifecycle methods.